### ISSUE TITLE
Supress (some) notifications / (UI)toggle to supress...?

### ISSUE THREAD

**Issue by raoulteeuwen** [abraunegg/onedrive#773]
Thank you for the work!

A question. When i lose connection (like when commuting from work to home etc), i often have the OneDrive client constantly popping up a stream of notifications. Not sure when this triggers.

Would love to get rid of those or (as i do love to know whether onedrive is running and in sync etc and would not want to simply supress all notifications all the time) be able to toggle it off/on.

Running onedrive v2.3.9-1 on Kubuntu 19.10 with Plasma 5.16.5.

**Comment by abraunegg** [comment]
@raoulteeuwen 
> Running onedrive v2.3.9-1

You are running a **VERY** old client version. Please update your client.

Please also refer to https://github.com/abraunegg/onedrive#reporting-issues when submitting issues:

1. Check the version of the application you are using onedrive --version and ensure that you are running either the latest release or built from master.

When raising an issue, please use the issue template: https://github.com/abraunegg/onedrive/issues/new?template=bug_report.md

**Comment by raoulteeuwen** [comment]
@abraunegg i'm sorry. I've just recently set up this config, and assumed the onedrive i had installed was new.

I'm now on "onedrive v2.3.13-7-gb81b969" and will see what that brings. 

Fyi: I have tried to follow your install instructions. With my level of knowledge, when arrived at "Build options" i did not know where, when or how to use those, so the last step i ended up doing is "sudo make install" .
